src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -324,12 +328,12 @@ internal static IConfiguration GetConfiguration(string settingsFile)
         /// </summary>
         /// <param name="csdl">The CSDL stream.</param>
         /// <returns>An OpenAPI document.</returns>
-        public static async Task<OpenApiDocument> ConvertCsdlToOpenApi(Stream csdl, string settingsFile = null)
+        public static async Task<OpenApiDocument> ConvertCsdlToOpenApi(Stream csdl, CancellationToken token, string settingsFile = null)
```

Comment:
conventions: make the cancellation token the last parameter, and make it optional as well.